src/chapters/more-on-functions/functions-as-arguments.rst

```diff
@@ -0,0 +1,285 @@
+Functions as Arguments
+======================
+
+You now know that *functions are data*, and therefore can be passed around just like other values. In particular, this means a function can be *passed to another function* as an argument. This turns out to be extremely useful. 
+
+Passing Functions as Arguments
+------------------------------
+
+Lots of JavaScript functions allow you pass a function as an argument when calling them. A function passed in this way can be called later, potentially multiple times. It allows the *function being called* to use the *function argument* to carry out its action.
```

Comment:
I think the first sentence is better now. I removed the first "function" since using "argument" and "pass" implicitly make it clear what we're talking about.

The second sentence is not specifically about callbacks, though callbacks are used in this way. It attempts to explain what happens to the function argument, and how it is used by the receiving function. The third sentence explains this better though, so I'll axe the second.